vmconf.py

```diff
@@ -13,4 +13,47 @@
 # limitations under the License.
 
 """Stores various configuration options and constants for Oppia-ml."""
-pass
+
+# The platform for the storage backend. This is used in the model-switching
+# code in core/platform.
+PLATFORM = 'gce'
+
+# Whether we should serve the development or production experience.
+# DEV_MODE should only be changed to False in the production environment.
+DEV_MODE = True
+
+# Default communication URL to be used to communicate with Oppia server.
+# This URL will be used in local development.
+DEFAULT_COMMUNICATION_URL = 'https://localhost'
+
+# Default communication PORT to be used to communicate with Oppia server.
+# This URL will be used in local development.
+DEFAULT_COMMUNICATION_PORT = 8181
+
+# Communication URL which is to be used in production environment to
+# communicate with Oppia server.
+SERVER_COMMUNICATION_URL = 'https://www.oppia.org'
+
+# Communication PORT which is to be used in production environment to
+# communicate with Oppia server.
+SERVER_COMMUNICATION_PORT = 80
+
+# Default ID which is to be used in local development environment.
+DEFAULT_VM_ID = 'vm_default'
+
+# Default shared key which is to be used in local development environment.
+DEFAULT_VM_SHARED_SECRET = '1a2b3c4e'
+
+# Name of metadata parameter which will contain ID of the VM in production
+# environment in GCE.
+METADATA_VM_ID_PARAM_NAME = 'vm_id'
+
+# Name of metadata parameter which will contain shared secret of the VM in
+# production environment in GCE.
+METADATA_SHARED_SECRET_PARAM_NAME = 'shared_secret_key'
+
+# Handler URL of Oppia which is used to retrieve jobs.
+FETCH_NEXT_JOB_REQUEST_HANDLER = '/fetchnextjobhandler'
```

Comment:
Can we document these interfaces on a wiki page somewhere so that there's a fixed canonical reference for the (narrow) API between the two servers? This is important; anything pertaining to the interface in either codebase should then adhere to that spec.